src/modules/getBullsAndCows.js

```diff
@@ -13,7 +13,23 @@
  * Example: { bulls: 1, cows: 2 }
  */
 function getBullsAndCows(userInput, numberToGuess) {
-  /* Write your code here */
+  const result = {
+    bulls: 0,
+    cows: 0,
+  };
+
+  const userNumber = userInput.toString();
+  const guessNumber = numberToGuess.toString();
+
+  for (let i = 0; i < userNumber.length; i++) {
+    if (userNumber[i] === guessNumber[i]) {
+      result.bulls++;
+    } else if (guessNumber.includes(userNumber[i])) {
+      result.cows++;
```

Comment:
The current logic for counting cows may overcount if a digit appears more than once in the `numberToGuess`. Implement a mechanism to track which digits have already been counted as cows to ensure accuracy.